plugins-ui/src/modules/core/components/ui/accordion/Accordion.tsx

```
@@ -0,0 +1,44 @@
+import { useState, ReactNode } from "react";
+import "./Accordion.css";
+import ChevronDown from "@/assets/ChevronDown.svg?react";
+import ChevronRight from "@/assets/ChevronRight.svg?react";
+import Button from "../button/Button";
+
+type AccordionProps = {
+  header: ReactNode;
+  expandButton: {
+    text: string;
+    style?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)